kubernetes/corefile-tool/README.md

```diff
@@ -0,0 +1,62 @@
+## Corefile-tool
+
+Corefile-tool is a simple command line tool which helps you to evaluate and migrate your CoreDNS Corefile Configuration.
+It is based on the [CoreDNS migration tool library](https://github.com/coredns/deployment/tree/master/kubernetes/migration).
+
+## Usage
+
+Use the following syntax to run the `corefile-tool` command:
+
+`corefile-tool [command] [flags]`
+
+where `command`, `flags` are:
+
+- `command`: The operation you want to perform. 
```

Comment:
Need to define what the valid commands are.  I think these are in the "Operations" section, but the terminology  is not consistent command/operation, so it's confusing.